modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -118,10 +118,15 @@
           writableStoreOverlay = lib.mkIf config.ghaf.development.debug.tools.enable "/nix/.rw-store";
 
           qemu = {
-            extraArgs = [
-              "-device"
-              "vhost-vsock-pci,guest-cid=${toString cfg.vsockCID}"
-            ];
+            extraArgs =
```

Comment:
You could use the qemu.nix for the qemu `extraArgs`, conditionally on the vhotplug being enabled?